cmd/server/assets/users/index.html

```diff
@@ -84,17 +86,31 @@ <h1>Users</h1>
           {{$pages := .pages}}
           <ul class="pagination">
             <li class="page-item {{if eq $pages.Previous -1}}disabled{{end}}"><a class="page-link"
-                href="?offset={{$pages.Previous}}">Previous</a>
+                href="?{{if $s}}search={{$s}}&{{end}}offset={{$pages.Previous}}">Previous</a>
```

Comment:
I'd argue there's no reason for this if statement. Throughout this file you check if $s exists, but given string's default empty state is empty string, there is no reason for these if statements.

Reply:
Alternatively, you can build this once at the top where you define `$s` as `$searchQuery` or something.

Reply:
I like @sethvargo 's idea to remove the annoying redundancy.
This is to avoid `users/search=&offset=25` situations with an empty, useless query param